contracts/MedianOracle.sol

```diff
@@ -56,17 +64,20 @@ contract MedianOracle is OwnableUpgradeable, IOracle {
      *                        pass before a report is usable
      * @param minimumProviders_ The minimum number of providers with valid
      *                          reports to consider the aggregate report valid.
+     * @param scalar_ The scaling factor applied on the result.
      */
     function init(
         uint256 reportExpirationTimeSec_,
         uint256 reportDelaySec_,
-        uint256 minimumProviders_
+        uint256 minimumProviders_,
+        uint256 scalar_
```

Comment:
Thinking about this more I think we need different scalar for different providers when using different sources for target price (we can do that in a later change though)